sandbox/modules/identity/src/main/java/org/opensearch/identity/IdentityPlugin.java

```diff
@@ -93,21 +100,31 @@ private static boolean isEnabled(final Settings settings) {
     }
 
     @Override
-    public UnaryOperator<RestHandler> getRestHandlerWrapper(final ThreadContext threadContext) {
-        if (!enabled) {
-            return (rh) -> rh;
-        }
-        return (rh) -> securityRestHandler.wrap(rh);
+    public List<RestHandler> getRestHandlers(
+        Settings settings,
+        RestController restController,
+        ClusterSettings clusterSettings,
+        IndexScopedSettings indexScopedSettings,
+        SettingsFilter settingsFilter,
+        IndexNameExpressionResolver indexNameExpressionResolver,
+        Supplier<DiscoveryNodes> nodesInCluster
+    ) {
+        final List<RestHandler> handlers = new ArrayList<>(3);
+        handlers.add(new RestPutPermissionAction());
+        // Add more handlers for future actions
+        return handlers;
     }
 
+    // register actions in this plugin
     @Override
-    public List<ActionFilter> getActionFilters() {
```

Comment:
Why is this removing `getActionFilters`?

Reply:
I meant to move it down, but appeared to overwrite it. I am adding it back in. Sorry about that.